app/src/main/java/com/github/se/gatherspot/ui/topLevelDestinations/Chats.kt

```diff
@@ -82,10 +86,10 @@ fun Chats(viewModel: ChatsListViewModel, nav: NavigationActions) {
         when {
           events.isEmpty() -> {
             Box(
-                modifier = Modifier.fillMaxWidth().padding(paddingValues),
-                contentAlignment = Alignment.TopStart) {
+                modifier = Modifier.fillMaxSize().padding(paddingValues),
+                contentAlignment = Alignment.Center) {
                   Text(
-                      text = "No chatting group found...",
+                      text = if (isOnline) "Loading..." else "Your device is currently offline.",
```

Comment:
"Loading..." --> "No chatting group found..."

Reply:
Sure, I will change that